packages/asl-projects/client/pages/sections/index.js

```diff
@@ -86,7 +86,13 @@ class Questions extends PureComponent {
               <Controls
                 onContinue={async () => {
                   await advance();
-                  window.location.reload();
+                  // get the URL path for establishments
+                  const isEstablishmentsSection = window.location.pathname.includes('/edit/establishments');
+                  const establishmentCare = values['establishments-care-conditions'];
+                  // only skip reload if on establishments section AND ecc is true
+                  if (!(isEstablishmentsSection && (establishmentCare === true || establishmentCare === 'true'))) {
+                    window.location.reload();
+                  }
```

Comment:
The comments are explaining what the code does which can be inferred from the code, what we need to include is why this is being done.
```suggestion
                  // When first loading the add additional establishment form, the page reload removes the default
                  // initial establishment. Since the refresh is for change badge synchronisation, we can safely skip
                  // it when adding the first additional establishment to a licence.
                  const isEstablishmentsSection = window.location.pathname.includes('/edit/establishments');
                  const establishmentCare = values['establishments-care-conditions'];
                  if (!(isEstablishmentsSection && (establishmentCare === true || establishmentCare === 'true'))) {
                    window.location.reload();
                  }
```